Index: git/src/buffers/ByteBuffer.cs
===================================================================
--- git.orig/src/buffers/ByteBuffer.cs	2020-04-04 20:41:07.801526430 +0200
+++ git/src/buffers/ByteBuffer.cs	2020-04-04 20:41:07.801526430 +0200
@@ -27,6 +27,8 @@
 using Bless.Tools;
 using Mono.Unix;
 
+using Range=Bless.Util.Range;
+
 namespace Bless.Buffers {
 
 ///<summary>
Index: git/src/tools/find/IFindStrategy.cs
===================================================================
--- git.orig/src/tools/find/IFindStrategy.cs	2020-04-04 20:41:07.801526430 +0200
+++ git/src/tools/find/IFindStrategy.cs	2020-04-04 20:41:07.801526430 +0200
@@ -23,6 +23,8 @@
 using Bless.Util;
 using Bless.Buffers;
 
+using Range=Bless.Util.Range;
+
 namespace Bless.Tools.Find {
 
 public interface IFindStrategy
@@ -40,4 +42,4 @@
 	bool Cancelled { get; set; }
 		}
 
-} // end namespace
\ No newline at end of file
+} // end namespace
Index: git/src/tools/find/IFinder.cs
===================================================================
--- git.orig/src/tools/find/IFinder.cs	2020-04-04 20:41:07.801526430 +0200
+++ git/src/tools/find/IFinder.cs	2020-04-04 20:41:07.801526430 +0200
@@ -23,6 +23,8 @@
 using Bless.Util;
 using Bless.Buffers;
 
+using Range=Bless.Util.Range;
+
 namespace Bless.Tools.Find {
 
 public delegate void FirstFindHandler();
@@ -41,4 +43,4 @@
 	IAsyncResult ReplaceAll(byte[] ba, AsyncCallback ac);
 }
 
-} // end namespace
\ No newline at end of file
+} // end namespace
Index: git/src/tools/find/FindOperations.cs
===================================================================
--- git.orig/src/tools/find/FindOperations.cs	2020-04-04 20:41:07.801526430 +0200
+++ git/src/tools/find/FindOperations.cs	2020-04-04 20:41:07.801526430 +0200
@@ -24,6 +24,8 @@
 using System.Threading;
 using Bless.Util;
 
+using Range=Bless.Util.Range;
+
 namespace Bless.Tools.Find {
 
 ///<summary>
Index: git/src/gui/areas/AreaGroup.cs
===================================================================
--- git.orig/src/gui/areas/AreaGroup.cs	2020-04-04 20:41:07.801526430 +0200
+++ git/src/gui/areas/AreaGroup.cs	2020-04-04 20:41:07.801526430 +0200
@@ -25,6 +25,8 @@
 using Bless.Util;
 using Bless.Gui.Drawers;
 
+using Range=Bless.Util.Range;
+
 namespace Bless.Gui.Areas
 {
 
Index: git/src/gui/DataBookFinder.cs
===================================================================
--- git.orig/src/gui/DataBookFinder.cs	2020-04-04 20:41:07.801526430 +0200
+++ git/src/gui/DataBookFinder.cs	2020-04-04 20:41:07.801526430 +0200
@@ -26,6 +26,8 @@
 using Bless.Buffers;
 using Bless.Tools.Find;
 
+using Range=Bless.Util.Range;
+
 namespace Bless.Gui {
 
 public class DataBookFinder : IFinder
