Index: git/bless-script.in
===================================================================
--- git.orig/bless-script.in	2019-05-01 10:10:21.558220546 +0200
+++ git/bless-script.in	2019-05-01 10:34:23.513075601 +0200
@@ -1,3 +1,3 @@
-#!@SHELL@
+#!/bin/sh
 
-mono "@blesssrcdir@/bin/bless.exe" "$@"
\ No newline at end of file
+mono "@blesssrcdir@/bin/bless.exe" "$@"
Index: git/data/help_script.tmpl.in
===================================================================
--- git.orig/data/help_script.tmpl.in	2019-05-01 10:10:21.566220669 +0200
+++ git/data/help_script.tmpl.in	2019-05-01 10:34:30.061288724 +0200
@@ -1,4 +1,4 @@
-#!@SHELL@
+#!/bin/sh
 
 # Script that loads help for Bless
 xml_help="@blessdocdir@/user/bless-manual.xml"
Index: git/src/bless-script.in
===================================================================
--- git.orig/src/bless-script.in	2019-05-01 10:10:21.602221220 +0200
+++ git/src/bless-script.in	2019-05-01 10:34:41.389648505 +0200
@@ -1,3 +1,3 @@
-#!@SHELL@
+#!/bin/sh
 
-mono "@blesslibdir@/@PACKAGE_NAME@/bless.exe" "$@"
\ No newline at end of file
+mono "@blesslibdir@/@PACKAGE_NAME@/bless.exe" "$@"
